src/util/createStore.js

```diff
@@ -5,9 +5,19 @@
 import { applyMiddleware, createStore, compose } from 'redux';
 import getPlatformMiddleware from '../middleware/epic';
 import { fetchQueries, mergeCookies } from '../util/fetchUtils';
+import getClickTracker from './clickTracking';
 
 const noopMiddleware = store => next => action => next(action);
 
+export const clickTrackEnhancer = createStore => (reducer, initialState, enhancer) => {
+	const store = createStore(reducer, initialState, enhancer);
+	const clickTracker = getClickTracker(store);
+	document.body.addEventListener('click', clickTracker);
+	document.body.addEventListener('change', clickTracker);
```

Comment:
Doesn't `click` also get fired for `change` events on input, select, and text areas? Would this send clickTracking data twice?